src/com/facebook/buck/cxx/CxxTest.java

```diff
@@ -83,7 +78,7 @@ public CxxTest(
       Tool executable,
       ImmutableMap<String, String> env,
       Supplier<ImmutableList<String>> args,
-      ImmutableSortedSet<? extends SourcePath> resources,
+      @SuppressWarnings("unused") ImmutableSortedSet<? extends SourcePath> resources,
```

Comment:
Could you remove it?

Reply:
Sure

Reply:
ant complains if I remove this though :(